CHANGELOG.md

```diff
@@ -3,10 +3,17 @@
 ## v46.0.0-SNAPSHOT - unreleased
 
 ### 🎁 New Features
+
 * `FilterChooser` has new `menuWidth` prop, allowing you to specify as width for the dropdown
   menu that is different from the control.
 
+### 💥 Breaking Changes
+
+* `LocalDate.toString()`, `LocalDate.toJSON()` and `LocalDate.valueOf()` all return a string with dashes: `YYYY-MM-DD`.  
+  Prior versions returned the string without dashes: `YYYYMMDD`.  Requires Hoist-Core v13.0.5 or later.
```

Comment:
if this is just for the parsing of the value in the admin activity/errors,  we could redo those to submitting to submit the old YYYYMMDD format temporarily, to avoid requiring taking a hoist core release to take this new hoist-react version

Reply:
The main point is that applications need to be warned that if they were depending on the format of dates being submitted to the server as being YYYYMMDD they will have to change to accommodate that.

Reply:
I've fixed the submits on the activity/error models and updated the changelog notice.